index.js

```diff
@@ -34,18 +38,31 @@ EncryptedField.prototype.vault = function(name) {
 
             previous = new Buffer(previous);
 
-            var iv = previous.slice(0, self._iv_length);
-            var content = previous.slice(self._iv_length, previous.length);
-            var decipher = crypto.createDecipheriv(self._algorithm, self.key, iv);
+            function decrypt(key) {
+                var iv = previous.slice(0, self._iv_length);
+                var content = previous.slice(self._iv_length, previous.length);
+                var decipher = crypto.createDecipheriv(self._algorithm, key, iv);
 
-            var json = decipher.update(content, undefined, 'utf8') + decipher.final('utf8');
-            return JSON.parse(json);
+                var json = decipher.update(content, undefined, 'utf8') + decipher.final('utf8');
+                return JSON.parse(json);
+            }
+
+            var keyCount = self.decryptionKeys.length;
+            for (var i = 0; i < keyCount; i++) {
+                try {
+                    return decrypt(self.decryptionKeys[i]);
+                } catch (error) {
```

Comment:
hmm, that just made me realise that the `JSON.parse` will throw if invalid JSON is returned, so we're probably ok here! (the "vault" is always encrypted JSON)